js-miniapp-sample/src/services/filedownload/actions.js

```diff
@@ -31,10 +31,12 @@ const requestDownloadFile = (
         return Promise.resolve(name);
       })
       .catch((error) => {
+        console.log('requestDownloadFile: CATCH BLOCK: ', error);
         dispatch({
           type: FILE_DOWNLOAD_FAILURE,
-          error,
+          error: error,
         });
+        throw error;
```

Comment:
I see try catch & throw in the same method

Reply:
This is as per the syntax we followed for throwing an error from catch block as in "[js-miniapp-sample/src/services/secure-storage/action.js](https://github.com/rakutentech/js-miniapp/blob/07e4a2a123550a31a86111b82050bece5e5a6406/js-miniapp-sample/src/services/secure-storage/actions.js#L45)".
Can you suggest me with this if we have a better way to handle this?